packages/components/src/Toast/Toast.tsx

```diff
@@ -45,8 +46,8 @@ export default function Toast({
       className={clsx(
         className,
         styles.toastDialog,
-        color === "success" && styles.colorSuccess,
-        color === "alert" && styles.colorAlert,
+        color === "success" && colorStyles.colorSuccess,
+        color === "alert" && colorStyles.colorAlert,
```

Comment:
I could also write a little utility to unduplicate these lines, but it's only two lines, so I'm gonna skip it for now.